### PR TITLE
Added RocketChat APB

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: rocketchat-apb/roles/deprovision-rocketchat-apb/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#############################################################################
+# Deprovision a service
+#############################################################################
+- k8s_v1_service:
+    name: rocketchat-apb
+    namespace: '{{ namespace }}'
+    state: absent
+
+
+#############################################################################
```
Comment (cfchase) [inline rocketchat-apb/roles/deprovision-rocketchat-apb/tasks/main.yml]: probably want to edit this comment now that we don't need the extra resource cleanup

**Comment by cfchase** [comment]
I think this might be time to bring up a question going forward.  Perhaps we should not be appending -apb on all the resources we create it's really a rocketchat app and service. The apb is just a deployment method, not really what we're creating.  I think it should probably be hidden as an implementation detail.  What do you and @fabianvf and @jmontleon think?

**Comment by fabianvf** [comment]
Makes sense to me, what do other deployment methods do? I feel like most use a file extension or something, which is sort of what we're doing with the -apb. Only place I really see it mattering is if someone were manually browsing dockerhub or something, with the current state of our org it would be tough to see what's an apb vs a dependency.

**Comment by jmontleon** [comment]
Looks good to me. I'm inclined to say ACK, except maybe dealing with getting an icon incorporated so we can use iconclass instead of imageurl.

**Comment by jmontleon** [comment]
ACK

**Pull request merged**